HelloWorlds/graphql/lib/models/book.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
```

Comment:
you could also do
```
const {Schema} = require('mongoose');
```
and you wouldn't need to require it on the next line, but up to your code style.